string  day             # The day this measurement is about.
float64 avg_temp        # Average temperature.
float64 max_temp        # Maximum reported temperature.
float64 min_temp        # Minimum reported temperature.
bool    big_difference  # Indicates if there is big difference between minimum
                        # and maximum tempereature.